solc.js

```diff
@@ -104,58 +105,52 @@ function makeSourcePathRelativeIfPossible(sourcePath) {
   for (const absolutePrefix of [absoluteBasePath].concat(absoluteIncludePaths)) {
     const relativeSourcePath = path.relative(absolutePrefix, absoluteSourcePath);
 
-    if (!relativeSourcePath.startsWith('../'))
-      return withUnixPathSeparators(relativeSourcePath);
+    if (!relativeSourcePath.startsWith('../')) { return withUnixPathSeparators(relativeSourcePath); }
   }
 
   // File is not located inside base path or include paths so use its absolute path.
   return withUnixPathSeparators(absoluteSourcePath);
 }
 
-function toFormattedJson(input) {
+function toFormattedJson (input) {
   return JSON.stringify(input, null, program.prettyJson ? 4 : 0);
 }
 
-function reformatJsonIfRequested(inputJson) {
+function reformatJsonIfRequested (inputJson) {
   return (program.prettyJson ? toFormattedJson(JSON.parse(inputJson)) : inputJson);
 }
 
-var callbacks = undefined
-if (options.basePath || !options.standardJson)
-  callbacks = {'import': readFileCallback};
+let callbacks;
+if (options.basePath || !options.standardJson) { callbacks = { import: readFileCallback }; }
```

Comment:
Oh, does it really reformat like this automatically?
Can't say I like this. Makes the code less readable in many cases.

Reply:
If we change the style I'd suggest in a different PR because that will apply to every single file. In this PR the only change really is enabling lint on `solcjs`.